packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -225,6 +241,33 @@ export function buildRootObject() {
         issuerTable.getIssuerRecordByIssuer(issuer).brand,
       getAmountMath,
       makeZCFMint,
+      addEmptySeat: keyword => {
+        const initialAllocation = harden({ [keyword]: undefined });
+        const proposal = harden({ want: { [keyword]: undefined } });
+        const { notifier, updater } = makeNotifierKit();
+        const zoeSeatAdminPromiseKit = makePromiseKit();
+
+        E(zoeInstanceAdmin)
+          .makeEmptySeat(keyword, initialAllocation, proposal)
+          .then(({ seatAdmin: zoeSeatAdmin, seatData }) => {
+            updateNotiferFrom(updater, seatData.notifier);
+            zoeSeatAdminPromiseKit.resolve(zoeSeatAdmin);
+          });
+
+        const seatData = harden({
+          proposal,
+          initialAllocation,
+          notifier,
+        });
+        const { zcfSeat, zcfSeatAdmin } = makeSeatAdmin(
+          allSeatStagings,
+          zoeSeatAdminPromiseKit.promise,
```

Comment:
Nice, since this is remote anyways, and we need to call E(zoeSeatAdmin), we can use a promise. Cool!